mongoimport/mongoimport.go

```diff
@@ -383,18 +410,14 @@ func (imp *MongoImport) importDocuments(inputReader InputReader) (numImported ui
 	}
 	log.Logvf(log.Info, "connected to node type: %v", imp.nodeType)
 
-	if err = imp.configureSession(session); err != nil {
-		return 0, fmt.Errorf("error configuring session: %v", err)
-	}
-
 	// drop the database if necessary
 	if imp.IngestOptions.Drop {
 		log.Logvf(log.Always, "dropping: %v.%v",
 			imp.ToolOptions.DB,
 			imp.ToolOptions.Collection)
-		collection := session.DB(imp.ToolOptions.DB).
-			C(imp.ToolOptions.Collection)
-		if err := collection.DropCollection(); err != nil {
+		collection := session.Database(imp.ToolOptions.DB).
+			Collection(imp.ToolOptions.Collection)
+		if err := collection.Drop(nil); err != nil {
```

Comment:
Does the following error check need to change?

Reply:
https://github.com/mongodb/mongo-go-driver/blob/master/mongo/collection.go#L1261 - seems like the driver doesn't propagate "ns not found" errors from Drop up to the user, so I changed this to remove the following error check.